docs/releases.md

```diff
@@ -30,69 +30,41 @@
 
 ## Master branch
```

Comment:
I would honestly delete the whole `Master branch` section; I don't think it belongs in the docs. We added it there to cover the large changes since 4.4.0, but now that we have a release out, it's served its purpose.

Reply:
Well it was there already before, it was just rarely updated :')

Reply:
I am fine with nuking it.

Reply:
Agreed. If it was kept up to date it might make sense, but we don't.